content/using-sagemaker-managed-spot-training/notebooks/configuring_libraries.md

```diff
@@ -0,0 +1,105 @@
+---
+title: "Configuring Libraries for Managed Spot Training"
+chapter: false
+weight: 30
+---
+
+Multiple libraries and SDKs such as the AWS Python SDK (boto3) and the Amazon SageMaker Python SDK support executing training jobs with Spot Instances. This workshop provides example notebooks that leverage multiple popular libraries and SDKs so that when working through the workshop you can choose to gain experience with those most relevant to your workload, or you can execute all of them as time permits to become familiar with various alternatives.
+
+## Available Notebooks
+
+* sagemaker-built-in-deepar - Example notebook for training and hosting a DeepAR model with the SageMaker Python SDK.
+* sagemaker-built-in-xgboost - Example notebook for single instance and distributed training of an XGBoost model with the AWS Python SDK (boto3).
+* sagemaker-built-in-image-classification - Example notebook for single instance training of an image classification model with the AWS Python SDK (boto3). 
+* sagemaker-built-in-object-detection - Example notebook for initial and incremental training of an object detection model with the SageMaker Python SDK.
```

Comment:
On this notebook, I had an issue when fetching the image at : https://images.pexels.com/photos/980382/pexels-photo-980382.jpeg. I can get access externally but the wget command from the notebook was getting a 404...

Reply:
I'll check this out and can push that fix through the spot-labs repo.